packages/backend/src/open_payments/payment/outgoing/lifecycle.ts

```diff
@@ -81,13 +81,24 @@ export async function handleSending(
     finalDebitAmount: maxDebitAmount,
     finalReceiveAmount: maxReceiveAmount
   })
-  deps.telemetry
-    ?.getOrCreateMetric('transactions_total', {
-      description: 'Count of funded transactions'
-    })
-    .add(1, {
-      source: deps.telemetry.getInstanceName()
-    })
+
+  if (deps.telemetry) {
+    deps.telemetry
+      .getOrCreateMetric('transactions_total', {
+        description: 'Count of funded transactions'
+      })
+      .add(1, {
+        source: deps.telemetry.getInstanceName()
+      })
+
+    deps.telemetry
+      .getOrCreateHistogramMetric('outgoing_payment_completion_time_ms', {
+        description: 'Time to complete a payment'
+      })
+      .record(Date.now() - payment.createdAt.getTime(), {
+        source: deps.telemetry.getInstanceName()
+      })
```

Comment:
@mkurapov (and anyone else) do you think these following bucket thresholds would be better?:

```ts
deps.telemetry
      .getOrCreateHistogramMetric('outgoing_payment_completion_time_ms', {
        description: 'Time to complete a payment',
        advice: {
          explicitBucketBoundaries: [
            100,
            200,
            300,
            400,
            500,
            600,
            700,
            800,
            900,
            1000,
            2000,
            5000,
            10000,
            Infinity
          ]
        }
```

The default buckets which it's currently using (as can be seen in the pr body screenshot) are 0, 5, 10, 25, 50, 75, 100, 250, 500, 750, 1000, 2500, 5000, 7500, 10000, +Inf. So the lower part of the range is too low (it will never complete in 0, 5, etc. ms). Bad bucket sizes make the distribution graphs less useful.

However, this would probably complicate things on grafana, as the `outgoing_payment_completion_time_ms` already exists in prometheus with the default buckets so I didn't want to just change it. I think we'd need to either add a new name or somehow delete this existing one in prometheus. 

@beniaminmunteanu wondering if you can speak to whether or not we could easily whipe the existing `outgoing_payment_completion_time_ms` metric in prometheus? I think I'd prefer that if we are going to set different bucket thresholds because this is the name we want to use and I dont want to pollute prometheus with unused metrics. Also, what is the retention policy on metrics (should be set in prometheus config somewhere I think). If we do rename I guess these unused ones will just drop off eventually.

Reply:
I think we can leave the default for now TBH, and then fine-tune them with more data

Reply:
Sounds good... I will probably need to change the name anyways since we're changing to time just the ilp pay part. I just dreaded a `outgoing_payment_completion_time_ms` and `outgoing_payment_completion_time_ms_v2` situation but I think at some point they might fall off from being unused (depends on our configuration though I guess).